storage/rocksdb/rdb_locking_iter.h

```diff
@@ -42,6 +42,7 @@ class LockingIterator : public rocksdb::Iterator {
   bool  m_valid;
 
   ulonglong *m_lock_count;
+  ulonglong m_self_lock_count;
```

Comment:
This variable needs to be initialized to 0. Otherwise, it causes random failures when `m_self_lock_count > max_lock_count`.

Reply:
You are right, I missed that when I extracted this patch from a larger changeset and then I only tested it in a debug build after that. I'll fix it.